README.markdown

```diff
@@ -347,3 +347,15 @@ If setting the `default` alias does not establish the node version in new shells
 [3]: https://travis-ci.org/creationix/nvm
 [Urchin]: https://github.com/scraperwiki/urchin
 [Fish]: http://fishshell.com
+
+## Uninstall
+
+```
+rm -rf $NVM_DIR ~/.npm ~/.bower && \
```

Comment:
I guess that is a bit hack and slash but I believe running the install script with ~/.npm or ~/.bower existing caused problems when I tried it. Also it is more of a deep clean even if its not nvm specific which is always what I prefer. 

Could be rephrased to: 

> 
```
rm -rf $NVM_DIR && \
```
*note:* you may also want to clear you ~/.npm and ~/.bower if you have further problems. 


The install script works reliably but if you encounter problems its nice to know how to get back to clean slate to start from scratch.